src/devices/bus/gameboy/mbc.cpp

```diff
@@ -1439,3 +1449,117 @@ void gb_rom_camera_device::write_ram(offs_t offset, uint8_t data)
 			m_ram[ram_bank_map[m_ram_bank] * 0x2000 + (offset & 0x1fff)] = data;
 	}
 }
+
+// Vast Fame
+
+uint8_t gb_rom_vfame_device::read_rom(offs_t offset)
+{
+	bool inchargestartbank = (m_seq_start_bank == 0 && offset < 0x4000) || (m_seq_start_bank == m_latch_bank2 && offset >= 0x4000);
+	
+	if (inchargestartbank && offset == m_seq_start_addr && m_seq_bytes_left == 0)
+	{
+		m_seq_bytes_left = m_seq_length;
+	}
+	
+	if(m_seq_bytes_left > 0 && offset < 0x8000)
+	{
+		m_seq_bytes_left--;
+		return m_seq[m_seq_length - m_seq_bytes_left - 1];
+	}
+	
+	if(m_shouldreplace && offset >= m_replace_start_addr && offset < 0x4000)
+	{
+		return m_rom[m_replace_src_bank * 0x4000 + (offset & 0x3fff)];
+	}
+
+	if (offset < 0x4000)
+		return m_rom[rom_bank_map[m_latch_bank] * 0x4000 + (offset & 0x3fff)];
+	else
+		return m_rom[rom_bank_map[m_latch_bank2] * 0x4000 + (offset & 0x3fff)];
```

Comment:
Since this last if/else is copy pasted from the parent class it can be replaced with `return gb_rom_mbc5_device::read_rom(offset);`